packages/database/src/modelClasses/OptionSet.js

```diff
@@ -10,6 +10,11 @@ import { RECORDS } from '../records';
 export class OptionSetRecord extends DatabaseRecord {
   static databaseRecord = RECORDS.OPTION_SET;
 
+  async options() {
+    const options = await this.otherModels.option.find({ option_set_id: this.id });
+    return options.sort((a, b) => a.sort_order - b.sort_order);
```

Comment:
I figure it doesn’t make that much sense to have a sort order and display things all out of whack, so decided to sort in the model here

Reply:
Nice, I love that this is on the model now